app/javascript/packs/hello_react.jsx

```diff
@@ -0,0 +1,26 @@
+// Run this example by adding <%= javascript_pack_tag 'hello_react' %> to the head of your layout file,
+// like app/views/layouts/application.html.erb. All it does is render <div>Hello React</div> at the bottom
+// of the page.
+
```

Comment:
Please get rid of this file (it's created automatically when you installed webpacker, but it's extra code you don't need.